internal/pgbackrest/config.go

```diff
@@ -398,12 +398,14 @@ func reloadCommand(name string) []string {
 	// volume and configuration file. When either changes, signal pgBackRest
 	// and print the observed timestamp.
 	//
-	// We send SIGTERM because the TLS server in pgBackRest 2.36 must be
-	// restarted to change. We filter by parent process to ignore the forked
-	// connection handlers. Their parent process is one because they are
-	// detached/orphaned from the server. The server parent process is zero
-	// because it is started by Kubernetes.
-	//
+	// We send SIGHUP because this allows the TLS server configuration to be
+	// reloaded in pgBackRest 2.37. We filter by parent process to ignore the forked
+	// connection handlers.
+	// - https://github.com/pgbackrest/pgbackrest/commit/7b3ea883c7c010aafbeb14d150d073a113b703e4
+
+	// Their parent process is one because they are detached/orphaned from the server.
```

Comment:
🤔 Is this part about being orphaned still true? The commit linked above removes a call to `forkDetach()`.

I don't recall how I would have caught sight of child processes. If they live long enough, you can see them in the following:

```console
$ kubectl exec sts/mycluster-repo-host -c pgbackrest -- ps axf -O ppid

    PID    PPID S TTY          TIME COMMAND
 210238       0 R ?        00:00:00 ps axf -O ppid
     35       0 S ?        00:00:12 pgbackrest-config -ceu monitor
     28       0 S ?        00:00:11 pgbackrest server-start
      1       0 S ?        00:00:00 /pause
```

Reply:
That's a good question, I'm not sure. I missed the removal of the `forkDetach()` call, but, in light of that, I am thinking it may no longer be the case. Do you think it would be better to remove that comment?